src/services/types.ts

```diff
@@ -276,7 +278,7 @@ namespace ts {
         getApplicableRefactors(fileName: string, positionOrRaneg: number | TextRange): ApplicableRefactorInfo[];
         getEditsForRefactor(fileName: string, formatOptions: FormatCodeSettings, positionOrRange: number | TextRange, refactorName: string, actionName: string): RefactorEditInfo | undefined;
 
-        getEmitOutput(fileName: string, emitOnlyDtsFiles?: boolean): EmitOutput;

```

Comment:
this is gonna be a breaking change. use overloads instead to avoid the breaking change.

Reply:
never mind, i know see that they are subtypes.

overloads should be better nevertheless.